### PR TITLE
Cleaned up README.md

### PR DESCRIPTION
**Pull request by Exsects**
- Cleaned up README.md 
- Renamed and updated the following Wiki pages:
  -  Installing FFMPEG for mbox
  -  Setting up a Bot Account on the Discord Developer Portal 
-  Fixed images not showing up correctly for Wiki pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,81 +13,78 @@ Uses SponsorBlock data licensed used under CC BY-NC-SA 4.0. More details: https:
...
-    - Click on Bot tab.
 
-    - Scroll down to Privileged Gateway Intents and turn on both PRESENCE INTENT and SERVER MEMBERS INTENT. [(image)](https://raw.githubusercontent.com/borisliao/mbox/master/images/install4.png)
-    
-    - Scroll back up to Build-A-Bot and copy token. [(image)](https://raw.githubusercontent.com/borisliao/mbox/master/images/install2.png)
+    ```bash
+    # For Windows users
+    python -m venv venv/
+    venv\Scripts\activate
+
```
Comment (Karan-Jain) [inline README.md]: Remove the / after the venv, for windows users .


And it doesnt need to be venv, I had originally put "venv name" to give devs a choice. 

The Venv is local and so its is something that is not going into the codebase. So devs can name it whatever they want

**Comment by Karan-Jain** [comment]
Consider adding a comment at the start of the readme -> use python instead of pythom3 for windows users.

This way we dont need to have a seperate python command for window and mac/linux

File: README.md [hunk 2]
```diff
@@ -13,81 +13,78 @@ Uses SponsorBlock data licensed used under CC BY-NC-SA 4.0. More details: https:
...
 
 ## Setup/Installation
-1. Install Python 3.7 or greater from https://www.python.org/
-    - Mac/Linux users, you might already have python 3 installed on your computer. Open your terminal and type `python`. If your command prompt says `Python 3.7` or higher you do not need to install from the website.
-    - Mac/Linux users, your `python` command might use Python 2, which is not supported. If this is the case use the command `python3` instead of `python` for the rest of this setup. 
-        
+1. Install required software.  
+    -  Install Python 3.7 or greater from https://www.python.org/
+    
+    *For Windows users*, if you do not bash already installed, you can download Git Bash [here]((https://git-scm.com/download/win))
```
Comment (borisliao) [inline README.md]: Git Bash is not necessary needed in windows, only git. This download comes with git bash but primarily is for installing git.

**Review by borisliao — CHANGES_REQUESTED** [review]

**Review by Karan-Jain — APPROVED** [review]

**Review by Karan-Jain — DISMISSED** [review]

**Review by AvJain — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -13,81 +13,76 @@ Uses SponsorBlock data licensed used under CC BY-NC-SA 4.0. More details: https:
...
+
+# For MacOS/Linux users
+python3 main.py debug
 ```
 
+*Instead of running the commands above in the terminal, you can use the [Run and Debug](https://code.visualstudio.com/docs/editor/debugging) feature already built into VSCode. The following profiles has been included within the project: Normal Mode, Debug Mode*
+
 ## Contributing
-Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+Pull requests are welcomed. For major changes, please open an issue first to discuss what you would like to change.
```
Comment (borisliao) [inline README.md]:
```suggestion
Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
```

**Review by borisliao — DISMISSED** [review]
Either works but welcome sounds better then welcomed.

**Review by borisliao — APPROVED** [review]

**Review by borisliao — APPROVED** [review]

**Review by AvJain — APPROVED** [review]
Changes look good

**Review by borisliao — APPROVED** [review]
Looks great! Nothing has changed!!

**Review by AvJain — APPROVED** [review]
Nice changes

**Pull request merged**